components/certificate-mgt/org.wso2.carbon.certificate.mgt.core/src/main/java/org/wso2/carbon/certificate/mgt/core/impl/CertificateGenerator.java

```diff
@@ -596,4 +568,31 @@ private ASN1Primitive toASN1Primitive(byte[] data) {
 
         return null;
     }
+
+    /**
+     * Get Signed certificate by parsing certificate.

```

Comment:
Revisit this particular comment and make it more meaningful. It is not quite clear which certificate was parsed in order to produce the output.